scripts/create-component/templates/component/src/components/{{componentName}}/use{{componentName}}.ts.hbs

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { makeMergeProps, resolveShorthandProps } from '@fluentui/react-compose/lib/next/index';
+import { {{componentName}}Props, {{componentName}}State } from './{{componentName}}.types';
+import { use{{componentName}}State } from './use{{componentName}}State';
+
+/**
+ * Consts listing which props are shorthand props.
+ */
+export const {{camelCase componentName}}ShorthandProps = ['loader', 'content'];
+
+const mergeProps = makeMergeProps({ deepMerge: {{camelCase componentName}}ShorthandProps });
+
+/**
+ * Given user props, returns state and render function for a {{componentName}}.
+ */
+export const use{{componentName}} = (props: {{componentName}}Props, ref: React.Ref<HTMLElement>, defaultProps?: {{componentName}}Props) => {
+  // Ensure that the `ref` prop can be used by other things (like useFocusRects) to refer to the root.
+  // NOTE: We are assuming refs should not mutate to undefined. Either they are passed or not.
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const resolvedRef = ref || React.useRef();
+  const state = mergeProps(
+    {
+      ref: resolvedRef,
+      as: 'div',
+      content: { as: 'span', children: props.children }
```

Comment:
remove `content`

Reply:
Can I suggest instead of removing it as it gives implementation guidance, possibly just have it commented. Also with the storybook / example this is useful to quickly test the initial setup and you can confirm that the component was properly generated. Please advise

Reply:
I suggested to remove because `content` is not a universal slot like `ref` and `as` where all components will have by default.

Reply:
Updated